packages/yew-macro/tests/html_macro/list-fail.stderr

```diff
@@ -46,11 +46,11 @@ error: expected an expression following this equals sign
 18 |     html! { <key=></> };
    |                 ^^
 
-error: this closing tag has no corresponding opening tag
-  --> $DIR/list-fail.rs:20:36
+error: the property value must be either a literal or enclosed in braces. Consider adding braces around your expression like this: {"key" . to_string()}
```

Comment:
'{"key" . to_string()}' 
The spacing caused by the Display impl of TokenStream is a niggle.
I honestly don't know of a good solution to this - I did see a crate to 'faithfully' display a TokenStream but seems way overkill for this one use case (and I don't even know if it works :P) 
Not sure if this is me just being super picky or whether we leave out the example so that the last sentence would read: 'Consider adding braces around your expression`. 
sorry @Xavientois should have noticed this before the other suggestion about spacing 😞

Reply:
Yeah, I was trying to figure out how to fix that issue.
Is [this](https://crates.io/crates/proc-macro-faithful-display) the crate you're referring to? The implementation is relatively small and contained to a single file, so for now, I can leave the example out, and then once this PR merges, I can see if there is a straightforward way to get it to display correctly

Reply:
Yep, that's the crate I found. 
I think without the example the message is clear enough, and when the expression is larger the example probably adds noise.

I've just been playing with some of the examples with this PR and here is just a snippet from the boids example but instead of using the macro I just manually expanded it to see what a larger expression would look like:
```rust
html! {
    <div class="settings">
        <Slider label="Number of Boids"
            min=1.0 max=600.0
            onchange=link.callback(move |value| {
                let mut settings = settings.clone();
                settings.boids = value as usize;
                Msg::ChangeSettings(settings)
            })
            value={settings.boids as f64}
        />
    // rest of the html block 
```
This returns the following error:
```bash
error: the property value must be either a literal or enclosed in braces. Consider adding braces around your expression like this: {link .
callback(move | value |
         {
             let mut settings = settings . clone() ; settings . boids = value
             as usize ; Msg :: ChangeSettings(settings)
         })}
   --> examples/boids/src/main.rs:122:30
    |
122 |                       onchange=link.callback(move |value| {
    |  ______________________________^
123 | |                         let mut settings = settings.clone();
124 | |                         settings.boids = value as usize;
125 | |                         Msg::ChangeSettings(settings)
126 | |                     })
    | |______________________^
```

Without adding the example:
```bash
error: the property value must be either a literal or enclosed in braces. Consider adding braces around your expression.
   --> examples/boids/src/main.rs:122:30
    |
122 |                       onchange=link.callback(move |value| {
    |  ______________________________^
123 | |                         let mut settings = settings.clone();
124 | |                         settings.boids = value as usize;
125 | |                         Msg::ChangeSettings(settings)
126 | |                     })
    | |______________________^
```
I think this is much nicer now that I've laid them out here. The message is clear about the error and the span tells me exactly where I should add those braces I've been told to consider! :D

Reply:
Done